checks/cloud/google/iam/no_folder_level_default_service_account_assignment_test.go

```diff
@@ -22,20 +22,14 @@ func TestCheckNoFolderLevelDefaultServiceAccountAssignment(t *testing.T) {
 		{
```

Comment:
Would we still need this test? I see some of the other Go bases tests have been removed.

Reply:
Only the tests that have been migrated to Rego have been deleted